overlays/overlay.nix

```diff
@@ -1,8 +1,19 @@
-{ inputs }:
+{ inputs, nixpkgs }:
 
 final: prev:
+let
+  pkgs = import nixpkgs {
+    system = "x86_64-linux";
+  };
 
-{
+  gitignore = pkgs.fetchFromGitHub { 
+    owner = "hercules-ci";
+    repo = "gitignore.nix";
+    rev = "637db329424fd7e46cf4185293b9cc8c88c95394";
+    sha256 = "sha256-HG2cCnktfHsKV0s4XW83gU3F57gaTljL9KNSuG6bnQs=";
+  };
```

Comment:
Still feel like this could be switched to using a flake in the inputs in `flake.nix`, then referring to package from there.

Reply:
Fixed this in `3b162b4`